Sources/SpeziLLM/Views/LLMChatView.swift

```diff
@@ -92,9 +95,13 @@ public struct LLMChatView<Session: LLMSession>: View {
     /// Creates a ``LLMChatView`` with a `Binding` of a ``LLMSession`` that provides developers with a basic chat view to interact with a Spezi LLM.
     ///
     /// - Parameters:
-    ///   - model: A `Binding` of a  ``LLMSession`` that contains the ready-to-use LLM to generate outputs based on user input.
-    public init(session: Binding<Session>) {
+    ///   - session: A `Binding` of a  ``LLMSession`` that contains the ready-to-use LLM to generate outputs based on user input.
+    ///   - exportFormat: An optional ``ChatView.ChatExportFormat`` that defines the format of the to-be-exported ``Chat``
+    ///         - Can be .pdf, .json, .text, or .none (default is .pdf)
+    ///         -  If .none, the chatview will present no export button in the toolbar
```

Comment:
Reference the `ChatView` via a single backtick

Reply:
It appears that replacing the '.' with a backslash does not fix the issue. Neither does replacing the expression with 'SpeziChat/ChatView/ChatExportFormat', '/ChatView/ChatExportFormat', or 'SpeziLLM/ChatView/ChatExportFormat'. Any advice here? Thanks so much for the feedback!

Reply:
What are you exactly trying to do? As stated above, one sadly isn't able to refer to types from other SPM packages via the double backtick notation. Therefore, please just use `ChatView/ChatExportFormat`, that's good enough 👍